manifests/g/Ghisler/TotalCommander/10.50/Ghisler.TotalCommander.locale.en-US.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate
+
+PackageIdentifier: Ghisler.TotalCommander
+PackageVersion: 10.50
+PackageLocale: en-US
+Publisher: Ghisler Software GmbH
+PackageName: Total Commander
+PackageUrl: https://www.ghisler.com
+License: Copyright (c) Christian Ghisler.
```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate